### ISSUE TITLE
incorrect placement of elements ?

### ISSUE THREAD

**Issue by maxim3007** [Andereoo/TkinterWeb#12]
Hi!
I've found that on some websites tkhtml renders page incorrectly:

url : http://w3.org/svg

HtmlFrame:

<img width="1679" alt="src" src="https://user-images.githubusercontent.com/76749786/121237748-0dfdee00-c8a0-11eb-86a1-9a51ac59722b.png">

Chrome:

<img width="1680" alt="src1" src="https://user-images.githubusercontent.com/76749786/121237814-1fdf9100-c8a0-11eb-8298-855828d3abf3.png">

Is it normal ??

**Comment by Andereoo** [comment]
Yes, unfortunately it is normal.

Tkhtml was discontinued a while ago, and doesn't support many modern CSS declarations.
Websites change as web standards evolve, but tkhtml hasn't had the developers to continue to support these new standards.
Tkhtml actually does support enough CSS to render http://w3.org/svg correctly, but since that website now relies on more 'modern' CSS, tkhtml doesn't know what to do with it. 

For example, https://upload.wikimedia.org/wikipedia/en/9/95/Tkhtml.PNG shows Tkhtml's rendering of the Wikipedia homepage a couple of years ago. If you visit https://en.wikipedia.org/wiki/Main_Page from TkinterWeb today, you'll notice that some parts of the page have been messed up.

Basically, if you altered the CSS code in http://w3.org/svg to rely on CSS 2 instead of CSS 4, the website would look perfectly fine.

**Comment by maxim3007** [comment]
is there any way to fix it without recompiling tkhtml?

**Comment by Andereoo** [comment]
Oh, I see. That is actually a brilliant idea! It would likely slow down page load times by a lot, but I'd say it's worth it. I only have to find a converter that would work. Any ideas?

**Comment by maxim3007** [comment]
I've viewed some css code and tested it with TkinterWeb and I think tkhtml has some problems with aligns and paddings

**Comment by maxim3007** [comment]
Hi again!
I've found css to scss converters (https://cssportal.com/css-to-scss/ , https://github.com/ortic/css2scss)
Maybe if write them in python they can help wit this issue but I am not sure

**Comment by maxim3007** [comment]
*with

**Comment by Andereoo** [comment]
I don't think the scss converter would work, we'd need to convert css3 to css2, not scss.

Also, do you mind uploading the code you were using to test Tkhtml?

**Comment by maxim3007** [comment]
Sorry , it is not my code and I've already deleted it. Sorry again

**Comment by Andereoo** [comment]
Don't worry about it! I was just curious. 

For now I'm going to close this, because I don't think that there is anything I can do from within this package to address the rendering inconsistencies. If I do find something helpful, I'll re-open this issue and let you know!
